.gitignore

```diff
@@ -0,0 +1,68 @@
+Makefile.in
+aclocal.m4
+autom4te.cache/
+compile
+config.guess
+config.h.in
+config.sub
+configure
+depcomp
+doc/Makefile.in
```

Comment:
Just `Makefile.in`. Remove this and any other occurrencies of Makefile.in in this file.